articles/anomaly-detection/guides/set-anomaly-detection-preferences.md

```diff
@@ -1,70 +1,81 @@
+Automated attack protection mitigates scripted attacks by detecting when a request is likely to be coming from a bot. These are sometimes called  [credential stuffing](/anomaly-detection/concepts/credential-stuffing) attacks. It is enabled by default for all connections. It provides a basic level of protection against certain attacks that adds very little friction to legitimate users. When such an attack is detected, it displays a CAPTCHA step in the login experience to eliminate bot and scripted traffic.
 
 ::: warning
-Auth0 strongly recommends that you **do not** set the `brute_force_protection` flag to `false` (effectively disabling brute-force protection for the connection), however if you do, you can change it back in the [Dashboard](${manage_url}/#/anomaly). 
+Auth0 strongly recommends that you **do not** disable **Automated Attack Protection**, however if you do, you can enable it in the [Dashboard](${manage_url}/#/anomaly).
 ::: 
 
-Limit the amount of signups and failed logins from a suspicious IP address. For more information, see [Brute-Force Protection Triggers and Actions](/anomaly-detection/references/brute-force-protection-triggers-actions).
+1. Go to [Dashboard > Anomaly Detection](${manage_url}/#/anomaly)
+2. Click on the **Automated Attack Protection** shield.
+3. Choose whether you wish to use the simple CAPTCHA provided by Auth0, or Google reCAPTCHA (requires external setup and registration). 
 
-1. Click on the **Brute-force Protection** shield. 
+    - If you choose simple CAPTCHA, you are done with set up.
 
-![Brute-Force Protection Shield](/media/articles/anomaly-detection/brute-force-shield.png)
+    ![Simple CAPTCHA](/media/articles/anomaly-detection/simple.png)
 
-2. Use the toggles to enable or disable actions for single or multiple user accounts. 
+    - If you choose Google's reCAPTCHA, enter the **Site Key** and **Secret** you obtain when you register your app with Google. 
 
-3. Add any IP addresses to the **Whitelist** field to avoid erroneously triggering the protection action.
+    ![Google reCAPTCHA](/media/articles/anomaly-detection/google.png)
 
-4. Click **Save** when you are finished.
+4. Click **Save**.
 
-## Breached password detection preferences
+## Brute-force Protection preferences
 
-Set preferences for breached password detection actions. For more information, see [Breached Password Detection Triggers and Actions](/anomaly-detection/references/breached-password-detection-triggers-actions).
+[Brute-force protection](/anomaly-detection/concepts/brute-force-protection) is enabled by default for all connections. Once enabled, you can customize the brute-force protection settings. You can limit the number of signups and failed logins from a suspicious IP address.
 
-1. Click on the **Breached-password Detection** shield.
+::: warning
+Auth0 strongly recommends that you **do not** disable **Brute-force Protection**, however if you do, you can enable it in the [Dashboard](${manage_url}/#/anomaly).
```

Comment:
[...] you can enable it back in the Dashboard.

Reply:
Fixed